po/common/common.pot

```diff
@@ -285,9 +285,9 @@ msgid ""
 "svg/donate-icon.svg\" alt=\"Donate to Open Food Facts\" /></"
 "a><p><<site_name>> is made by a non-profit association, independent from the "
 "industry. It is made for all, by all, and it is funded by all. You can "
-"support our work by <a href=\"https://world.openfoodfacts.org/donate-to-open-food-facts\">donating to Open Food Facts</a> and "
+"support our work by <a href=\"https://world.openfoodfacts.org/donate-to-open-food-facts\" style="text-decoration:underline;">donating to Open Food Facts</a> and "
```

Comment:
We have many other links like this one that are in the middle of other text. I'm thinking a better idea would be to change the CSS to underline links inside paragraph tags.

To discuss: https://github.com/openfoodfacts/openfoodfacts-server/issues/5074

Reply:
Yes, I'll start working on it.